types/ion-rangeslider/util/parse_settings.py

```diff
@@ -0,0 +1,155 @@
+from IPython import embed
```

Comment:
What is the purpose of adding this file to DefinitelyTyped?

Reply:
Automatically parse and create types directly from ion.rangeSlider documentation. I'd be open to remove this from DefinitelyTyped, and instead save this in another repository. If so, would I can add a link in a comment instead.